### PR TITLE
TeamSonarr.Sonarr version 3.0.6.1196

### PR DESCRIPTION
**Pull request by alteredpark**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17765)

### CODE REVIEW

File: manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.6.1196
+PackageLocale: en-US
+Publisher: Team.Sonarr
```
Comment (OfficialEsco) [inline manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.locale.en-US.yaml:7]:
```suggestion
Publisher: Team Sonarr
```

File: manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.6.1196
```
Comment (OfficialEsco) [inline manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.installer.yaml:5]:
```suggestion
PackageVersion: 3.0
```

File: manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.6.1196
```
Comment (OfficialEsco) [inline manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.0
```

File: manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.6.1196
```
Comment (OfficialEsco) [inline manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.yaml:5]:
```suggestion
PackageVersion: 3.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
This is required Information that needs to match what Windows Sees.
Remember to change the folder and file names.

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
I see you created a issue over at Sonarr https://github.com/Sonarr/Sonarr/issues/4542
This version of Sonarr will not get approved as `3.0.6.1196` since it will cause issues with `winget upgrade`, either they have to rerelease the installer or you have to wait for them to fix your issue in the next release.

**Comment by OfficialEsco** [comment]
https://github.com/Sonarr/Sonarr/issues/4542

**Comment by mikeparkie** [comment]
@OfficialEsco correct and thanks for the comments. I'll close this req and re-raise once/if they update the installer 👍

File: manifests/t/TeamSonarr/Sonarr/3.0/TeamSonarr.Sonarr.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0
+PackageLocale: en-US
+PackageUrl: https://sonarr.tv/
+Publisher: TeamSonarr
```
Comment (OfficialEsco) [inline manifests/t/TeamSonarr/Sonarr/3.0/TeamSonarr.Sonarr.locale.en-US.yaml:6]:
```suggestion
Publisher: Team Sonarr
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]
🎉

**Pull request merged**